usagov/vote-gov-drupal
# Vote-988 ->  php lint errors

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[Vote-988](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-998&assignee=611272e0eb7aef00694b8a97)

## Description (optional)
This PR will address PHP lint errors that were happing in the pipeline and fix errors that were appearing.

## Deployment and testing (required, if applicable)
### Testing steps
1. Scroll to the bottom and check that checks are passing and not showing any PHP linting errors
